PC/layout/support/pymanager.py

```diff
@@ -39,7 +39,6 @@ def calculate_install_json(ns, *, for_embed=False, for_test=False):
     DISPLAY_NAME = "Python"
     TAG_SUFFIX = ""
     ALIAS_PREFIX = "python"
-    ALIAS_WPREFIX = "pythonw"
```

Comment:
This is meant to be used... looks like 132 should use it instead of combining with `ALIAS_PREFIX`, and I think the intent was to add an `if not prefix: continue` inside the loop (so that a package with no aliases, such as the embeddable distro, wouldn't try to add any - guessing line 60 also should be duplicated for `WPREFIX`).

Reply:
See #133120